apis/v1beta1/types.go

```diff
@@ -265,6 +268,15 @@ type PCIDeviceSpec struct {
 	CustomLabel string `json:"customLabel,omitempty"`
 }
 
+// VGPUSpec defines virtual machine's VGPU configuration
+type VGPUSpec struct {
+	// ProfileName is the ProfileName of a virtual machine's vGPU, in string.
+	// Defaults to the eponymous property value in the template from which the
+	// virtual machine is cloned.
+	// +kubebuilder:validation:Required
+	ProfileName string `json:"profileName,omitempty"`
```

Comment:
we handle this in `reconcilePCIDevices`. Should this instead of being a separate field next to PciDevices be integrated into PciDevices instead?

Reply:
That makes sense. I'll try to integrate the vgpu profile name into PCIDeviceSpec.

Reply:
Worked well in manual test with only vgpu profilename and omitting deviceID and vendorID.